doc/man1/pkeyparam.pod

```diff
@@ -18,8 +18,8 @@ B<openssl> B<pkeyparam>
 
 =head1 DESCRIPTION
 
-The B<pkey> command processes public or private keys. They can be converted
-between various forms and their components printed out.
+The B<pkeyparam> command processes public key algorithm parameters.
+They can be checked for correctness and their components printed out.
 
```

Comment:
I removed the sentence "They can be converted between various forms" because it does not seem to be true, only PEM format is supported. However, I am wondering what sense it makes to be able to read and then write the parameters in (the same) PEM format? Did I overlook something here? What is the use case of reading _and_ writing the same format?

Reply:
It would make sense if we supported DER format as well...